<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaisonForm</class>
 <widget class="QWidget" name="MaisonForm">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1239</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Ajouter une Maison</string>
  </property>
  <property name="styleSheet">
   <string>background-color: white; color: black;</string>
  </property>
  <widget class="QLabel" name="labelCle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Clé :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCle">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelType">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Type :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxType">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <item>
    <property name="text">
     <string>LOCATION</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>VENTE</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelStanding">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>151</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Standing :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxStanding">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Haut</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Moyen</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bas</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelToilettes">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>151</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Toilettes :</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxToilettes">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>330</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelChambres">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>151</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Chambres :</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxChambres">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>400</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelDescription">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>100</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Description :</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditDescription">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>150</y>
     <width>300</width>
     <height>160</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAjouterPhoto">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>330</y>
     <width>300</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Ajouter photos</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfoAjouterPhoto">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>385</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Aucune photo ajoutée</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonValider">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>500</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitre">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>560</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ajouter une Maison</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
